README.md

```diff
@@ -26,15 +26,26 @@ make install-dev
 ```
 
 3. Install third-party dependencies:
+
```

Comment:
You can delete this paragraph and we can just keep the `make install-third-party-external`

Reply:
Ok, done in https://github.com/kscalelabs/sim/commit/29a8d0e551312b089ee8447fc7cfb300fa031698 .